### PR TITLE
Move almost everything to OwlBot

### PR DESCRIPTION
**Pull request by jskeet**
This requires an internal CL to be merged first, then I'll rerun generateprojects.sh to delete synth files and generate OwlBot configurations - I don't want to do that until we're ready to merge as it wouldn't rebase cleanly.

### CODE REVIEW

File: tools/Google.Cloud.Tools.Common/ApiMetadata.cs [hunk 1]
```diff
@@ -147,7 +147,7 @@ public AutoGeneratorType DetermineAutoGeneratorType(string apiDirectory)
             // Currently, default to just "synthtool if anything". Once we have confidence that
```
Comment (amanda-tarafa) [inline tools/Google.Cloud.Tools.Common/ApiMetadata.cs:147]: nit: update this comment

File: tools/Google.Cloud.Tools.ReleaseManager/ValidateOwlBotCommand.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            var apis = catalog.Apis.Where(x => x.DetermineAutoGeneratorType() == AutoGeneratorType.OwlBot).ToList();
+            var ungenerated = apis.Where(api => !IsGenerated(api, googleapisGen)).ToList();
+            foreach (var api in ungenerated)
+            {
+                Console.WriteLine($"Ungenerated API: {api.Id}");
+            }
+            Console.WriteLine($"Total OwlBot APIs: {apis.Count}; ungenerated APIs: {ungenerated.Count}");
+        }
+
+        private bool IsGenerated(ApiMetadata api, string googleapisGen)
```
Comment (amanda-tarafa) [inline tools/Google.Cloud.Tools.ReleaseManager/ValidateOwlBotCommand.cs:50]: If I understand this right, it just needs to have been generated once. We can't check for correct re-generation, right?

**Review by amanda-tarafa — APPROVED** [review]
LGTM

File: tools/Google.Cloud.Tools.Common/ApiMetadata.cs (hunk 1 above)
Comment (jskeet) [reply tools/Google.Cloud.Tools.Common/ApiMetadata.cs]: Done.

File: tools/Google.Cloud.Tools.ReleaseManager/ValidateOwlBotCommand.cs (hunk 2 above)
Comment (jskeet) [reply tools/Google.Cloud.Tools.ReleaseManager/ValidateOwlBotCommand.cs:50]: That's correct. But I think that's likely to be good enough, to be honest.

**Comment by jskeet** [comment]
(Everything is now being generated, so I'll merge on green.)

**Pull request merged**